### PR TITLE
Unit Test bodypart/organ things, better handling for edge cases

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Stolen from @Time-Green 

## Changelog

:cl: Time-Green
fix: Odd cases where bodyparts and organs end up outside of mobs are handled.
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
TODO: the same thing for bodyparts

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -1444,3 +1445,9 @@
 	if(!H?.can_ingest_reagents)
 		return FALSE
 	return TRUE
+
+/mob/living/carbon/dropItemToGround(obj/item/I, force, silent, invdrop)
+	if(I && HAS_TRAIT(I, TRAIT_INSIDE_BODY))
+		stack_trace("Something tried to drop an organ or bodypart that isn't allowed to be dropped")
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/carbon.dm:1451]: Not a bad idea because fuck a lot of code be doing this

**Review by Time-Green — APPROVED** [review]
